sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableAttributeBindingProvider.cs

```diff
@@ -39,6 +39,12 @@ private IBinding TryCreate(BindingProviderContext context)
                 return null;
             }
 
+            // ParameterBindingData and ParameterBindingData[] are bound by the next binding
```

Comment:
Not very clear what _next binding_ means. What if later on someone adds another binding function in between this and ParameterBinding one. 

May be we can say, _bound by separate binding methods `BindToInput<T>()`specific to those types_ or something on similar lines.